### DISCUSSION TITLE
How to create a PSBT?

### DISCUSSION THREAD

**Discussion by adityanithariya** [paulmillr/micro-btc-signer#39] [Q&A] [answered]
I wanted to do batch transactions, for which PSBTs should be created, so that it can be used with any of the wallet available.
Docs aren't very helpful and understandable. Needed to understand example given here: https://docs.xverse.app/sats-connect/creating-psbts
Illustrate using examples for better understanding!

**Comment by paulmillr** [comment]
The readme is filled with 20 examples. What does it lack, in your opinion? Do you know btc, in general? Because the library requires some basic btc knowledge.

**Reply by adityanithariya** [reply]
It may happen that due to lack of complete knowledge about BTC I'm not able to understand docs. Will you please suggest me any reference for knowledge about it as I need to integrate batch transactions with my app urgently. Even if I'll not be able to understand, please give me your contact for dm or should I ask you in this thread?

**Reply by paulmillr** [reply]
What's batch transactions? Can you describe the exact use-case in this thread, step-by-step, of what you're trying to do?

**Reply by adityanithariya** [reply]
To have multiple outputs of a transaction is termed as batch transactions.
I'll be integrating it with unisat wallet to send BTC to invoice address for BRC-20 Inscription and another output as minting fee to my wallet.
For integration, I need to create PSBT hex and then pass it to signTransaction function provided by unisat wallet window API

**Reply by paulmillr** [reply]
We should support multiple outputs: https://github.com/paulmillr/scure-btc-signer#outputs

**Reply by adityanithariya** [reply]
Will you please give me a complete live example as I am lacking in some parts of docs?

**Reply by adityanithariya** [reply]
Can you please provide your discord so that It'll be easy for me to communicate with you?

**Reply by paulmillr** [reply]
I can't provide discord tech support to all users. This is free project. 

We can communicate here.

**Comment by paulmillr** [comment, marked as answer]
Ordinals got easier with 1.3.0 https://github.com/paulmillr/scure-btc-signer/releases/tag/1.3.0
